resources/js/Pages/Home.vue

```diff
@@ -74,8 +74,11 @@
             splitInput: function(): Array<string> {
                 return this.form.itemsInput.split( '\n' );
             },
+            sanitizeArray: function(): Array<string> {
+                return this.splitInput().filter(x => x);
```

Comment:
Ah, this is a clever way to do it, nicely done @guergana! 

I think we can also use a more explicit notation here, it might be more code, but I think it could be worth it. Otherwise, this will also filter out values like `0` (which are not allowed anyhow due to validation, but perhaps it's just best to make it more explicit)

Reply:
Ah, yes, my bad, I forgot.

Reply:
It's done. :)

Reply:
coolio